src/android/io/sqlc/SQLitePlugin.java

```diff
@@ -361,8 +367,10 @@ private boolean deleteDatabaseNow(String dbname) {
 
         public void run() {
             try {
+                logger.logWarning("Opening database (" +  + this.dbkey.hashCode() + ")", "SQLite");
```

Comment:
You have a typo here. Please remove the duplicated + on the message.

Reply:
Fixed